pkg/fleetautoscalers/controller.go

```diff
@@ -115,7 +115,7 @@ func (c *Controller) Run(workers int, stop <-chan struct{}) error {
 		return err
 	}
 
-	c.baseLogger.Info("Wait for cache sync")
+	c.baseLogger.Debug("Wait for cache sync")
```

Comment:
I'm wondering if we want to move the info=>debug logging items to a separate PR?

I'm just wondering if we want to come up with some rules of thumb on what should be `info and what should be `debug`? @pooneh-m @roberthbailey what do you think? I've no idea if there is prior art here?

Just thinking long term, and also for consistency?

Reply:
Rules of thumb sound good, but we should discuss them in an issue (so they aren't lost in this PR) and document them somewhere. 

I've created https://github.com/googleforgames/agones/issues/1223.

Reply:
https://reflectoring.io/logging-levels/ has some suggestions. :) I don't see an easy way of describing logging level other that use your judgement and get a code review for a second opinion. My judgement is if I delete a log would it reduce my confidence that the system is doing what is expected, if so make it Info, otherwise Debug.